Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2999,6 +3000,15 @@ extension BrowserViewController: IntroViewControllerDelegate {
             }
         }
     }
+    
+    func introViewControllerShouldAskForNotifications(_ introViewController: IntroViewController) -> Bool {
+        // Only include the slide if Leanplum has decided it needs to run and if the current locale is en_US/GB/CA/AU/HK/SG
+        return Leanplum.hasStarted() && Locale.current.identifier.startsWith("en_")
```

Comment:
Yes, @Sdaswani this delegate needs a better check. Probably something like `LeanplumIntegration.shared.shouldEnableForCurrentLocale` or so. Something that does not talk to the LP servers.